daheinze-oscheer/integers/adder.go

```diff
@@ -0,0 +1,5 @@
+package integers
```

Comment:
Please add a package description:

```suggestion
// Package integers contains code that runs operations on integers, such as add.
package integers
```